treeherder/auth/backends.py

```diff
@@ -1,10 +1,16 @@
+import json
 import logging
 import re
+import time
 
 from django.contrib.auth.models import User
 from django.core.exceptions import ObjectDoesNotExist
-from rest_framework.reverse import reverse
-from taskcluster import Auth
```

Comment:
This makes the taskcluster python package no longer used, so it and its subdependencies can be removed from `requirements/common.txt` 😃 (I love being able to reduce the number of deps - particularly in this case since the Python 3 only deps of taskcluster-py are very noisy for pyup updates!)